apps/mobile-wallet/locales/en-US/translation.json

```diff
@@ -360,5 +360,10 @@
   "Error while copying address": "Error while copying address",
   "This address is not valid": "This address is not valid",
   "A number is expected": "A number is expected",
-  "View full size": "View full size"
+  "View full size": "View full size",
+  "Disclaimer": "Disclaimer",
+  "Buy": "Buy",
+  "Could not find a receiving address": "Could not find a receiving address",
+  "Cannot proceed because no address has been found to receive the funds.": "Cannot proceed because no address has been found to receive the funds.",
```

Comment:
We don't need these 2 strings anymore, it's better to remove them to not add extra workload for the translators.

Reply:
Removed